server/src/main/java/org/elasticsearch/rest/action/cat/AbstractCatAction.java

```diff
@@ -76,6 +84,7 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
         }
     }
 
+    private String grepFilter;
```

Comment:
These are singletons so you'd have to put this somewhere else or parse after rendering. You can use `responseParams` so the request will run.